src/helpers/firebase.ts

```diff
@@ -0,0 +1,129 @@
+import snapshot from '@snapshot-labs/snapshot.js';
+import { initializeApp, cert, ServiceAccount } from 'firebase-admin/app';
+import { getMessaging } from 'firebase-admin/messaging';
+
+import db from './mysql';
+import { getProposal } from './proposal';
+
+import serviceAccount from '../../firebase.json';
+
+const firebaseApp = initializeApp({
+  credential: cert(serviceAccount as ServiceAccount)
+});
+const messaging = getMessaging(firebaseApp);
+const hubURL = process.env.HUB_URL || 'https://hub.snapshot.org';
+
+const getUserSpaces = async user => {
```

Comment:
Could you use this format for all the functions and add types to it?
```suggestion
async function getUserSpaces(user: string) {
```